Dimming with Volume Buttons

kian-shirazi opened the pull request:
This feature allows the user to control the brightness of the flashlight right after the volume buttons are used to turn it on. The default is to give a 2 second window to make these adjustments, this is adjustable in the UI via settings.

kian-shirazi committed:
Added the ability to adjust the brightness level using the volume buttons after using the double-volume button shortcut to activate the flashlight.

The interval is user-adjustable in the UI.

cyb3rko commented:
Thanks for your contribution, I like the idea.  
Before merging, I would like to make some adjustments:

- allow option only if volume buttons service is activated, otherwise it does not have any effect
- disable by default
- change icon to something time related

kian-shirazi committed:
Apply requested changes for volume dimming feature

- Disable by default

- Require volume buttons service to be active

- Update icon to time-related ic_timer

kian-shirazi committed:
Fix build error by removing undefined localeFilters block

xdkaplan commented:
This is a feature I would greatly appreciate. I almost exclusively use the volume button shortcut and would appreciate a way to control brightness. Suggestion: volume keys during this short window after activating via volume buttons service modify the Preferred Level. This way, it is 'saved' between uses. Example:

Am in a airplane or dark room. Need very dim flash to not be rude. Will probably need this several times over a few hours. Will later be in a regular space and may need 'full' strength flash.
